Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.h

```diff
@@ -106,11 +116,36 @@ class KeyboardMouse : public Core::Device
 
 private:
   void UpdateCursorInput();
+  void Generate_Octagon_Points(POINT octagon_points[8]);
```

Comment:
Consider using std::array instead of plain arrays, since their size can be enforced at compile-time, ditto below where applicable

Reply:
I can definitely use std::array<> :)